internal/api/instrumentation/probes.go

```diff
@@ -14,6 +14,9 @@ import (
 )
 
 const (
+	v1 = "v1"
+	v2 = "v2"
```

Comment:
You might consider moving these into the utils code/module.  You might also think about making these an enum so that you can't mistype the version number.  Something like this:

```go
type ApiVersion int

const (
	v1 Season = 1
	v2       = 2
)

func (v ApiVersion) String() string {
	switch v {
	case v1:
		return "v1"
	case v2:
		return "v2"
	}
	return "unknown"
}
```

I'm probably over engineering things here ^^ (and overly complicating it).  But whats the fun of engineering something if we can't over engineer it??

Reply:
lool agreed!  I like it!  :smile: